src/test/resources/names.pan

```diff
@@ -0,0 +1,12 @@
+object template names;
+
+include 'quattor/types/metadata';
+# wait for https://github.com/quattor/template-library-core/pull/127
+#bind '/metadata' = structure_metadata;
+
+prefix "/metadata/template/branch";
+"name" = "mybranch";
+"type" = "production";
```

Comment:
i absolutely have no idea yet what the type field is for :smile: but it's mandatory, so i just guessed what it could have meant. i'll just change this to `"type" = "sandbox";` then?